backend/stakepoold/server.go

```diff
@@ -454,8 +473,9 @@ func (ctx *appContext) processWinningTickets(wt WinningTicketsForBlock) {
 			} else {
 				loserCount++
 			}
-			log.Infof("winning ticket %v msa %v duration %v: %v",
-				w.ticket, w.msa, w.duration, w.err)
+			log.Infof("winning ticket %v msa %v duration %v (%v + %v [+ %v]): %v",
+				w.ticket, w.msa, w.duration, w.signDuration, w.sendDuration,
+				w.duration-w.signDuration-w.sendDuration, w.err)
```

Comment:
Will probably want to remove w.duration-w.signDuration-w.sendDuration in the future since so far it just looks like  the difference is precision/rounding.